src/main/java/org/cliclient/CliClient.java

```diff
@@ -34,27 +47,78 @@ public class CliClient {
 
     private static final Scanner scanner = new Scanner(System.in);
 
+    private static String networkRequestType = "POST";
+
+    // TODO: this is very hacky, properly handle this later!
+    private static String userSpecifiedFileName = "fibbonacci";
 
     public static void main(String[] args) {
-        System.out.println("Starting the compute job client...");
-        // Starting compute job with initial JSON
+        if (args.length == 0) {
+            System.out.println("Starting the compute job client...");
 
-        while (true) {
+            System.out.println("Set Network Request Type (POST/GRPC), POST by default");
+            String response = scanner.nextLine();
+            if ("GRPC".equalsIgnoreCase(response)) {
+                CliClient.networkRequestType = "GRPC";
+            }
 
-            List<Integer> userInput = getUserInputArr();
-            int[] userinputArr = userInput.stream().mapToInt(i -> i).toArray();
+            System.out.println("Set file name (Don't specify file type!): ");
```

Comment:
We'll definitely want to specify if it's locally or server side and make the images available server side at those endpoint filenames.